erpnext/accounts/doctype/sales_invoice_item/sales_invoice_item.json

```diff
@@ -281,7 +281,8 @@
    "fieldname": "discount_amount",
    "fieldtype": "Currency",
    "label": "Discount Amount",
-   "options": "currency"
+   "options": "currency",
+   "read_only": 1
```

Comment:
@bhattdevarsh   is this intentional because after that user not able to enter the amount in this field